libs/ui/src/lib/form-message/form-message.component.stories.ts

```diff
@@ -0,0 +1,56 @@
+import { Story, Meta } from '@storybook/angular';
+
+import { FormMessageComponent } from './form-message.component';
+
+export default {
+  title: 'Angular/Form/FormMessage',
+  component: FormMessageComponent,
+  args: {
+    display: true,
+    severity: 'warning',
+    message: 'Error',
+    label: '',
+  },
+} as Meta<FormMessageComponent>;
+
+const Template: Story<FormMessageComponent> = (args: FormMessageComponent) => ({
+  props: args,
+  /* template */
+  template: `
+     <cvi-ng-track [gap]="4">
+      <cvi-ng-form-item label="Input field"
+                        htmlId="input-id">
+        <cvi-ng-input [placeholder]="placeholder"
```

Comment:
Placeholder value is not set as an `arg` and therefore appears as `undefined`:
![image](https://github.com/e-gov/cvi/assets/1833029/047b24a4-cd5b-4992-ad1b-b82c1fd93be2)

We can perhaps remove the prop from the story template since it's not relevant for the story.